lib/pure/complex.nim

```diff
@@ -81,6 +81,13 @@ func abs2*[T](z: Complex[T]): T =
   ## that is the squared distance from (0, 0) to `z`.
   ## This is more efficient than `abs(z) ^ 2`.
   result = z.re * z.re + z.im * z.im
+  
+func sgn*[T](z: Complex[T]): Complex[T] =
```

Comment:
Why not `sign`?
```suggestion
func sign*[T](z: Complex[T]): Complex[T] =
```

Reply:
@Varriount because `sgn` is more common, it'll be more consistent, e.g. https://nim-lang.org/docs/math.html#sgn%2CT

Reply:
Oh well. I'd prefer for them to both be `sign`, but I see your point.